### ISSUE TITLE
ebay-combobox: floated label sometimes becomes stuck in inline position

### ISSUE THREAD

**Issue by ianmcburnie** [eBay/ebayui-core#1722]
# Bug Report

### Scenario 1

1. Set a default value in the textbox
2. Notice label is stuck in inline position (label tag still has `floating-label__label--inline`)

### Scenario 2

1. Click on textbox
3. Enter some text
4. Clear text
5. Enter some text
6. Notice label is stuck in inline position and obscuring the text

## Workaround

Clicking out of textbox and then back in, seems to reset the label to floating position.

## Notes

On the Skin side it looks like we are missing stories for combobox under floating label.

## Screenshots

![Screen Shot 2022-08-04 at 2 45 41 PM](https://user-images.githubusercontent.com/38065/182958506-e8881789-7b48-45df-a9f0-432b3434e643.png)

**Comment by agliga** [comment]
I think this is an issue with markup floating label. Seems like removing the value isn't being honored for combobox
